deploy/packages/parallel-libraries.yaml

```diff
@@ -35,7 +35,7 @@ packages:
       - mpi
       - python
     specs:
-      - py-mpi4py@3.0.0
+      - py-mpi4py
```

Comment:
Just to explain the change dropping the version.If we specify here and Spack has a different default version then, in case it's used for dependencies too, two versions end up being installed without need. I feel in that case we should leave the version to the specified, either in dependency spec, the package recipe, or packages.yaml.

Reply:
this looks fine.